### PR TITLE
Fix accessibility issues relating to landmarks

### PR DESCRIPTION
**Pull request by heatherpekrul**
Using the Axe Coconut Chrome extension, I discovered several accessibility issues relating to landmarks on the vuejs.org website. Fixing these will make it easier for those with screen readers to use the web site.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "vuejs.org",
   "private": true,
   "hexo": {
-    "version": "3.8.0"
+    "version": "3.9.0"
```
Comment (phanan) [inline package.json]: Can you reverse the changes made to this file?

Comment (heatherpekrul) [reply package.json]: Sure - I've reverted this

File: package.json [hunk 2]
```diff
@@ -29,4 +29,4 @@
     "hoek": "^6.1.2",
     "request": "^2.85.0"
   }
-}
+}
```
Comment (phanan) [inline package.json]: Please revert this change as well.

Comment (heatherpekrul) [reply package.json]: Done.

**Comment by phanan** [comment]
Looks great, thanks a lot!
